app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -326,6 +326,7 @@ ensogl::define_endpoints_2! {
         /// background. In edit mode, the whole node area is considered non-active.
         background_press         (),
         expression               (ImString),
+        port_expression          (span_tree::Crumbs,ImString),
```

Comment:
1. space missing after `,`
2. Why the name was changed from `expression`? This is node expression, right? If it is not a node expression anymore, this needs docs.

Reply:
It is not changed. Both are there. One is a full expression update, the other is partial (under specified node expression span). I have added the docs to differentiate between those two.